src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -2605,8 +2604,7 @@ CreateShellTableOnWorkers(Oid relationId)
 	List *commandList = list_make1(DISABLE_DDL_PROPAGATION);
 
 	IncludeSequenceDefaults includeSequenceDefaults = WORKER_NEXTVAL_SEQUENCE_DEFAULTS;
-	IncludeIdentities includeIdentityDefaults =
```

Comment:
Please remove all references to `INCLUDE_IDENTITY_AS_SEQUENCE_DEFAULTS` in the code (and comments), as we should not need it anymore?